### DISCUSSION TITLE
In "@azure/communication-react": "1.5.0" npm version , ChatComposite component does not have filesharing options

### DISCUSSION THREAD

**Discussion by nnagakarthikeyan** [Azure/communication-ui-library#2828] [Q&A] [answered]
I am using "@azure/communication-react": "1.5.0" npm version. In this version we do not have "fileSharing" property as part of options attribute in ChatComposite Component. In this case how file sharing can be enabled for ChatComposite component in "1.5.0" version of communication ui Library ?

**Comment by JamesBurnside** [comment, marked as answer]
Currently the feature is currently in public preview and this is only available in our beta-packages

```sh
npm i @azure/communication-react@1.5.1-beta.2 @azure/communication-calling@1.12.0-beta.2
```

Looks like the quickstart incorrectly does not highlight this (I have filed an issue to get this fixed https://github.com/MicrosoftDocs/azure-docs/issues/107969), but once you have those packages installed you should be good to follow this to get it up and running: https://learn.microsoft.com/en-us/azure/communication-services/tutorials/file-sharing-tutorial
